src/javaharness/java/arcs/android/demo/service/ArcsService.java

```diff
@@ -30,6 +30,7 @@
 // TODO: this is generic Arcs service class, move outside demo.
 public class ArcsService extends IntentService {
   public static final String INTENT_REFERENCE_ID_FIELD = "intent_reference_id";
+  public static final String INTENT_EVENTLET_FIELD = "intent_eventlet";
```

Comment:
Is there documentation somewhere on what an "eventlet" is? This probably isn't the place for it, but I don't really understand what it means.

Reply:
renamed to event-data here, but it is called eventlet everywhere in runtime code, and technically contains all data associated with the ui event. i don't think there is documentation...